.env

```diff
@@ -29,3 +29,8 @@ ESPRESSO_DEMO_L1_PROVIDER=$ESPRESSO_SEQUENCER_L1_PROVIDER
 ESPRESSO_DEMO_HOTSHOT_ADDRESS=$ESPRESSO_SEQUENCER_HOTSHOT_ADDRESS
 ESPRESSO_DEMO_ROLLUP_ADDRESS=0xe7f1725e7734ce288f8367e1bb143e90bb3f0512
 ESPRESSO_DEMO_ROLLUP_MNEMONIC="test test test test test test test test test test test junk"
+
+# Foundry
+MNEMONIC="test test test test test test test test test test test junk"
+SEPOLIA_RPC_URL=
```

Comment:
Do we need these empty env variables?

Reply:
I don't know a better place to put them. From experience the `.env` file is much more likely to be consulted than the README. Maybe we can add some comments to explain what they do.